src/main/java/org/broadinstitute/hellbender/utils/read/markduplicates/OpticalDuplicateFinder.java

```diff
@@ -185,6 +188,14 @@ protected final int rapidParseInt(final String input) {
         return val;
     }
 
+    public boolean isOpticalDuplicate(PhysicalLocation a, PhysicalLocation b) {
+      return a.getReadGroup() == b.getReadGroup() &&
+          a.getTile() == b.getTile() &&
+          // There used to not be an abs here, not sure why.
```

Comment:
Sure, how should we do that?

On Wed, Jun 24, 2015 at 4:19 PM, Nils Homer <notifications@github.com>
wrote:

> In
> src/main/java/org/broadinstitute/hellbender/utils/read/markduplicates/OpticalDuplicateFinder.java
> <https://github.com/broadinstitute/hellbender/pull/587#discussion_r33197981>
> :
>
> > @@ -185,6 +188,14 @@ protected final int rapidParseInt(final String input) {
> >          return val;
> >      }
> >
> > +    public boolean isOpticalDuplicate(PhysicalLocation a, PhysicalLocation b) {
> > +      return a.getReadGroup() == b.getReadGroup() &&
> > +          a.getTile() == b.getTile() &&
> > +          // There used to not be an abs here, not sure why.
>
> @tovanadler <https://github.com/tovanadler> can we chat off-github?
> easier for me to explain there.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/broadinstitute/hellbender/pull/587/files#r33197981>.
>